app/mailers/communicart_mailer.rb

```diff
@@ -7,9 +7,11 @@ class CommunicartMailer < ActionMailer::Base
   add_template_helper ClientHelper
 
 
-  def proposal_notification_email(to_email, approval, show_approval_actions=true)
+  def proposal_notification_email(to_email, approval,
+                                  show_approval_actions=true, alert_partial=nil)
```

Comment:
Maybe we should use [keyword arguments](https://robots.thoughtbot.com/ruby-2-keyword-arguments)? Would make the calling more readable.